src/lang/en/seat.php

```diff
@@ -53,7 +53,7 @@
     'summary'               => 'Summary',
     'quantity'              => 'Quantity',
     'volume'                => 'Volume',
-    'group'                 => 'Group',
+    'group'                 => 'Group|Groups',
```

Comment:
Unfortunately, changing a translation to have pluralization support is going to require you to update everywhere else where this translation key is used. An example of where this change breaks other places where `trans()` is used and not `trans_choice()` can be seen in the contracts details view:

<img width="554" alt="screen shot 2018-05-14 at 07 30 21" src="https://user-images.githubusercontent.com/1148127/39979771-b2ef39c2-5748-11e8-81c4-58ff6b99a2ea.png">

I too prefer to use the translation support for this. Could you update the application wide instances with this change too?

Reply:
A quick `grep` through the sources show this list:

<img width="944" alt="screen shot 2018-05-14 at 07 36 35" src="https://user-images.githubusercontent.com/1148127/39979930-909c8b6c-5749-11e8-81e4-11ddbebc042f.png">